lib/ansible/modules/cloud/amazon/iam_role.py

```diff
@@ -288,7 +288,7 @@ def create_or_update_role(connection, module):
     role = get_role(connection, module, params['RoleName'])
 
     role['attached_policies'] = get_attached_policy_list(connection, module, params['RoleName'])
-    module.exit_json(changed=changed, iam_role=camel_dict_to_snake_dict(role))
+    module.exit_json(changed=changed, **camel_dict_to_snake_dict(role))
```

Comment:
Can you add back the initial return to maintain backwards compatibility so we don't break people's stuff?

Reply:
👍  done